Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.World.cpp

```diff
@@ -1548,6 +1548,43 @@ int CLuaFunctionDefs::ResetVehiclesLODDistance(lua_State* luaVM)
     return 1;
 }
 
+int CLuaFunctionDefs::GetPedsLODDistance(lua_State* luaVM) 
+{ 
+    float fPedsDistance; 
+ 
+    g_pGame->GetSettings()->GetPedsLODDistance(fPedsDistance); 
+    lua_pushnumber(luaVM, fPedsDistance); 
+    return 1; 
+}
+ 
+int CLuaFunctionDefs::SetPedsLODDistance(lua_State* luaVM) 
+{ 
+    float fPedsDistance; 
+ 
+    CScriptArgReader argStream(luaVM); 
+    argStream.ReadNumber(fPedsDistance); 
+    fPedsDistance = Clamp(0.0f, fPedsDistance, 500.0f); 
```

Comment:
Any reason for not using the MAX_PEDS_LOD_DISTANCE constant ?
(I can see that `SetVehiclesLODDistance` is not doing that either, genuine question)

Reply:
other namespace, i tested and not worked

Reply:
Please, place the line `fPedsDistance = Clamp(0.0f, fPedsDistance, 500.0f); `  after this `if (!argStream.HasErrors()) {`